play/src/front/Stores/MediaStore.ts

```diff
@@ -243,12 +254,20 @@ export const videoConstraintStore = createVideoConstraintStore();
  * A store that contains video constraints.
  */
 function createAudioConstraintStore() {
-    const { subscribe, update } = writable({
+    const initialConstraints: MediaTrackConstraints = {
         //TODO: make these values configurable in the game settings menu and store them in localstorage
         autoGainControl: false,
         echoCancellation: true,
         noiseSuppression: true,
-    } as boolean | MediaTrackConstraints);
+    } as MediaTrackConstraints;
```

Comment:
Should not delete this "as" by a "satisfies" ?

Reply:
:+1: